tests/cypress/e2e/results.cy.js

```diff
@@ -653,33 +666,32 @@ config?.pages?.forEach((page, _i) => {
 						if (!config?.selectors?.sidebar?.facetWrapper || !config?.selectors?.sidebar?.facetTitle || !config?.selectors?.sidebar?.facetOption) {
 							this.skip();
 						} else {
-							cy.get(`${config.selectors.sidebar.facetWrapper}`)
-								.each((facet, idx) => {
-									if (idx == 0) {
-										// click on an option in facet and ensure urlManager contains new state
-										const facetListOption = facet.find(config.selectors.sidebar.facetOption)[0];
-										if (facetListOption) {
-											cy.get(facetListOption).click({ force: true });
-										}
+							const listFacet = store.facets.filter((facet) => facet.display === 'list')[0];
+							if (!listFacet) this.skip();
+
+							cy.get(`${config.selectors.sidebar.facetWrapper}`).each((facet) => {
+								// find matching facet in dom
+								const title = facet.find(config.selectors.sidebar.facetTitle);
+								if (listFacet.label.trim() === title.text().trim()) {
+									if (listFacet.collapsed) {
+										// toggle visibility if collapsed
+										listFacet.toggleCollapse();
```

Comment:
Since this test is testing the "clear all" functionality - if we wanted to simplify this test we could apply a facet via the store to bypass the DOM entirely.